enterprise/utils.py

```diff
@@ -42,10 +42,7 @@
 except ImportError:
     configuration_helpers = None
 
-try:
-    from openedx.core.djangoapps.catalog.models import CatalogIntegration
-except ImportError:
-    CatalogIntegration = None
+from enterprise.api_client.enterprise_catalog import EnterpriseCatalogApiClient
```

Comment:
so the issue seems to be that importing this class has a cascade effect of importing `JwtLmsApiClient` which is what causes the import error. you will want to copy/paste the try/except ImportError logic that was used for CatalogIntegration above.